### ISSUE TITLE
Class 'Luthfi\FormField\FormFieldServiceProvider' not found

### ISSUE THREAD

**Issue by diorz38** [nafiesl/SimpleCrudGenerator#6]
when install this package, got an error:
```
In ProviderRepository.php line 208:

  Class 'Luthfi\FormField\FormFieldServiceProvider' not found
```
Command that i use:
```
λ composer require luthfi/simple-crud-generator --dev
Using version ^1.2 for luthfi/simple-crud-generator
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 3 installs, 0 updates, 0 removals
  - Installing symfony/dom-crawler (v4.1.7): Downloading (100%)
  - Installing laravel/browser-kit-testing (v4.1.0): Downloading (100%)
Downloading (100%)
  - Installing luthfi/simple-crud-generator (1.2.7): Downloading (100%)
Writing lock file
Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover
Discovered Package: nesbot/carbon
Discovered Package: fideloper/proxy
Discovered Package: laravel/tinker
Discovered Package: beyondcode/laravel-dump-server
Discovered Package: nunomaduro/collision
Discovered Package: luthfi/simple-crud-generator
Package manifest generated successfully.
```

try to create CRUD:
```
λ php artisan make:crud-simple Vehicle

In ProviderRepository.php line 208:

  Class 'Luthfi\FormField\FormFieldServiceProvider' not found
```
what do i wrong?
Laravel version: 5.7

**Comment by diorz38** [comment]
install it from existing project, i will try from fresh laravel 5.7, thx for super fast respond

**Comment by nafiesl** [comment]
This service provider `Luthfi\FormField\FormFieldServiceProvider`, might be defined on your `config/app.php` file (or anywhere on your project), but the `luthfi/formfield` package is not installed or has been removed.

Sure, your welcome. Glad to know that you are using the package and watching this repository. Thanks.

**Comment by diorz38** [comment]
I try to install with fresh laravel 5.5 its show an error:
```
E:\laragon\www\restApi
λ composer require luthfi/simple-crud-generator --dev
Using version ^1.2 for luthfi/simple-crud-generator
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - laravel/browser-kit-testing v4.1.0 requires phpunit/phpunit ^7.0 -> satisfiable by phpunit/phpunit[7.0.0, 7.0.1, 7.0.2, 7.0.3, 7.1.0, 7.1.1, 7.1.2, 7.1.3, 7.1.4, 7.1.5, 7.2.0, 7.2.1, 7.2.2, 7.2.3, 7.2.4, 7.2.5, 7.2.6, 7.2.7, 7.3.0, 7.3.1, 7.3.2, 7.3.3, 7.3.4, 7.3.5, 7.4.0, 7.4.1, 7.4.2, 7.4.3, 7.4.4, 7.4.x-dev, 7.5.x-dev] but these conflict with your requirements or minimum-stability.
    - laravel/browser-kit-testing v4.0.2 requires phpunit/phpunit ~7.0 -> satisfiable by phpunit/phpunit[7.0.0, 7.0.1, 7.0.2, 7.0.3, 7.1.0, 7.1.1, 7.1.2, 7.1.3, 7.1.4, 7.1.5, 7.2.0, 7.2.1, 7.2.2, 7.2.3, 7.2.4, 7.2.5, 7.2.6, 7.2.7, 7.3.0, 7.3.1, 7.3.2, 7.3.3, 7.3.4, 7.3.5, 7.4.0, 7.4.1, 7.4.2, 7.4.3, 7.4.4, 7.4.x-dev, 7.5.x-dev] but these conflict with your requirements or minimum-stability.
    - laravel/browser-kit-testing v4.0.1 requires phpunit/phpunit ~7.0 -> satisfiable by phpunit/phpunit[7.0.0, 7.0.1, 7.0.2, 7.0.3, 7.1.0, 7.1.1, 7.1.2, 7.1.3, 7.1.4, 7.1.5, 7.2.0, 7.2.1, 7.2.2, 7.2.3, 7.2.4, 7.2.5, 7.2.6, 7.2.7, 7.3.0, 7.3.1, 7.3.2, 7.3.3, 7.3.4, 7.3.5, 7.4.0, 7.4.1, 7.4.2, 7.4.3, 7.4.4, 7.4.x-dev, 7.5.x-dev] but these conflict with your requirements or minimum-stability.
    - laravel/browser-kit-testing v4.0.0 requires phpunit/phpunit ~7.0 -> satisfiable by phpunit/phpunit[7.0.0, 7.0.1, 7.0.2, 7.0.3, 7.1.0, 7.1.1, 7.1.2, 7.1.3, 7.1.4, 7.1.5, 7.2.0, 7.2.1, 7.2.2, 7.2.3, 7.2.4, 7.2.5, 7.2.6, 7.2.7, 7.3.0, 7.3.1, 7.3.2, 7.3.3, 7.3.4, 7.3.5, 7.4.0, 7.4.1, 7.4.2, 7.4.3, 7.4.4, 7.4.x-dev, 7.5.x-dev] but these conflict with your requirements or minimum-stability.
    - Conclusion: don't install luthfi/simple-crud-generator 1.2.8
    - Conclusion: don't install luthfi/simple-crud-generator 1.2.7
    - Conclusion: don't install luthfi/simple-crud-generator 1.2.6
    - Conclusion: don't install luthfi/simple-crud-generator 1.2.5
    - Conclusion: don't install luthfi/simple-crud-generator 1.2.4
    - Conclusion: don't install luthfi/simple-crud-generator 1.2.3
    - Conclusion: don't install luthfi/simple-crud-generator 1.2.2
    - Conclusion: don't install luthfi/simple-crud-generator 1.2.1
    - Conclusion: remove symfony/css-selector v4.1.7
    - Installation request for luthfi/simple-crud-generator ^1.2 -> satisfiable by luthfi/simple-crud-generator[1.2.0, 1.2.1, 1.2.2, 1.2.3, 1.2.4, 1.2.5, 1.2.6, 1.2.7, 1.2.8].
    - Conclusion: don't install symfony/css-selector v4.1.7
    - luthfi/simple-crud-generator 1.2.0 requires laravel/browser-kit-testing ^2.0 || ^4.0 -> satisfiable by laravel/browser-kit-testing[v2.0.0, v2.0.1, v4.0.0, v4.0.1, v4.0.2, v4.1.0].
    - laravel/browser-kit-testing v2.0.0 requires symfony/css-selector ~3.1 -> satisfiable by symfony/css-selector[v3.1.0, v3.1.1, v3.1.10, v3.1.2, v3.1.3, v3.1.4, v3.1.5, v3.1.6, v3.1.7, v3.1.8, v3.1.9, v3.2.0, v3.2.1, v3.2.10, v3.2.11, v3.2.12, v3.2.13, v3.2.14, v3.2.2, v3.2.3, v3.2.4, v3.2.5, v3.2.6, v3.2.7, v3.2.8, v3.2.9, v3.3.0, v3.3.1, v3.3.10, v3.3.11, v3.3.12, v3.3.13, v3.3.14, v3.3.15, v3.3.16, v3.3.17, v3.3.18, v3.3.2, v3.3.3, v3.3.4, v3.3.5, v3.3.6, v3.3.7, v3.3.8, v3.3.9, v3.4.0, v3.4.1, v3.4.10, v3.4.11, v3.4.12, v3.4.13, v3.4.14, v3.4.15, v3.4.16, v3.4.17, v3.4.18, v3.4.2, v3.4.3, v3.4.4, v3.4.5, v3.4.6, v3.4.7, v3.4.8, v3.4.9].
    - laravel/browser-kit-testing v2.0.1 requires symfony/css-selector ~3.1 -> satisfiable by symfony/css-selector[v3.1.0, v3.1.1, v3.1.10, v3.1.2, v3.1.3, v3.1.4, v3.1.5, v3.1.6, v3.1.7, v3.1.8, v3.1.9, v3.2.0, v3.2.1, v3.2.10, v3.2.11, v3.2.12, v3.2.13, v3.2.14, v3.2.2, v3.2.3, v3.2.4, v3.2.5, v3.2.6, v3.2.7, v3.2.8, v3.2.9, v3.3.0, v3.3.1, v3.3.10, v3.3.11, v3.3.12, v3.3.13, v3.3.14, v3.3.15, v3.3.16, v3.3.17, v3.3.18, v3.3.2, v3.3.3, v3.3.4, v3.3.5, v3.3.6, v3.3.7, v3.3.8, v3.3.9, v3.4.0, v3.4.1, v3.4.10, v3.4.11, v3.4.12, v3.4.13, v3.4.14, v3.4.15, v3.4.16, v3.4.17, v3.4.18, v3.4.2, v3.4.3, v3.4.4, v3.4.5, v3.4.6, v3.4.7, v3.4.8, v3.4.9].
    - Can only install one of: symfony/css-selector[v3.1.0, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.1.1, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.1.10, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.1.2, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.1.3, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.1.4, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.1.5, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.1.6, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.1.7, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.1.8, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.1.9, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.2.0, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.2.1, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.2.10, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.2.11, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.2.12, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.2.13, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.2.14, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.2.2, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.2.3, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.2.4, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.2.5, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.2.6, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.2.7, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.2.8, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.2.9, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.3.0, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.3.1, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.3.10, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.3.11, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.3.12, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.3.13, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.3.14, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.3.15, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.3.16, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.3.17, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.3.18, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.3.2, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.3.3, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.3.4, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.3.5, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.3.6, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.3.7, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.3.8, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.3.9, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.4.0, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.4.1, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.4.10, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.4.11, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.4.12, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.4.13, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.4.14, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.4.15, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.4.16, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.4.17, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.4.18, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.4.2, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.4.3, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.4.4, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.4.5, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.4.6, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.4.7, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.4.8, v4.1.7].
    - Can only install one of: symfony/css-selector[v3.4.9, v4.1.7].
    - Installation request for symfony/css-selector (locked at v4.1.7) -> satisfiable by symfony/css-selector[v4.1.7].


Installation failed, reverting ./composer.json to its original content.
```

but its not get an error when added  "luthfi/simple-crud-generator":  "^1.2" to composer.json manually

**Comment by nafiesl** [comment]
I have try to reproduce this error.

- I got same error after install via `composer require` on laravel 5.5 on php 7.2.
- I also got it worked after add the package name manually in `composer.json` file, than run `composer update`.

I found the problem, on **laravel 5.5 on php 7.2** :
- with composer require, composer trying to get laravel/browser-kit-testing v4.1.* which is only compatible with laravel ^5.6.
- with package name addition manually and composer update, composer gets laravel/browser-kit-testing v2.0.1 which is still compatible with laravel 5.5.

I think this issue will not get fixed. But we can **update the readme.md** on installation procedure for laravel 5.5 on php 7.2 (or rather since 7.1).

Thank you very much @diorz38 for reporting this issue. I really appreciate your attention.
